src/services/user.service.js

```diff
@@ -0,0 +1,29 @@
+import { User } from '../database/models';
+
+class UserService {
+  static createUser = async (data) => {
+    const user = await User.create(data);
+    return user;
+  };
+
+  static findUser = async (searchParams) => {
+    const user = await User.findOne({ where: searchParams });
+
+    if (user) {
+      return user;
+    }
+
+    throw new Error('User not found');
```

Comment:
We don't always want to throw an error for a user not found, so do the response with a 404 on the API level(in the controller). Throwing this error will cause the server to respond with a 404 based on your code and this might not be the intent